Code/GraphMol/MolStandardize/Tautomer.cpp

```diff
@@ -12,6 +12,7 @@
 #include <GraphMol/MolStandardize/FragmentCatalog/FragmentCatalogUtils.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
 #include <GraphMol/SmilesParse/SmilesWrite.h>
+#include <GraphMol/SmilesParse/SmartsWrite.h>
```

Comment:
It looks like this isn’t really used ( moltosmarts is commented out )

Reply:
good catch. fixed